source/fe/fe_tools.cc

```diff
@@ -1135,7 +1134,7 @@ namespace FETools
       Triangulation<dim,spacedim> tr;
       GridGenerator::hyper_cube (tr, 0, 1);
 
-      FEValues<dim,spacedim> coarse (mapping, fe, q_fine,
+      FEValues<dim,spacedim> coarse (StaticMappingQ1<dim,spacedim>::mapping, fe, q_fine,
```

Comment:
Same here, I wouldn't insert the mapping at all since we have that as a default.